nulib/meadow
# Replace ex_aws_s3 fork with latest supported version now that the fork has merged

# Summary 
We forked `ex_aws_s3` because of an issue with copying object tags, but the fork was merged in https://github.com/ex-aws/ex_aws_s3/pull/147

# Specific Changes in this PR
- Switch back to `ex_aws_s3` from the forked version

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
- Basically just make sure that S3 file operations still work (copying to preservation, etc).
- Make sure the preservation object's tags to make sure they are copied along with the object itself (the reason for the initial fork that has since been merged)

Also please let developers know if there are any special instructions to test this in the development environment. 

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
